docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+---
+title: Entra ID Integration
+description: Describes how to set up the Teleport Entra ID integration in Teleport.
+labels:
+ - how-to
+ - identity-governance
+---
+
+Entra ID integration enables the following features in Teleport:
+
+1. **Single sign-on (SSO):** Configures Teleport authentication with Entra ID as an
+   identity provider.
+2. **User sync:** Imports Entra ID users as Teleport users.
+3. **Group sync:** Imports Entra ID groups as Teleport
+   Access Lists. 
+4. **Application sync (Optional):** Imports Entra ID applications. 
```

Comment:
We do not import Applications into teleport. We only import them to access graph.
As opposed to Okta, we only import groups & users into Teleport